test/OpenTelemetry.Instrumentation.AWS.Tests/TestAWSClientInstrumentation.cs

```diff
@@ -169,8 +169,7 @@ public void TestSQSSendMessageSuccessful()
         {
             var sqs = new AmazonSQSClient(new AnonymousAWSCredentials(), RegionEndpoint.USEast1);
             string requestId = @"fakerequ-esti-dfak-ereq-uestidfakere";
-            string dummyResponse = "<?xml version=\"1.0\" encoding=\"UTF-8\"?>\n" +
-                                   "<ReceiveMessageResponse>SomeDummyResponse</ReceiveMessageResponse>";
+            string dummyResponse = "{}";
```

Comment:
This change is because AWSSDK.SQS 3.7.300 includes a change to default to use [the SQS JSON protocol instead of XML](https://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSDeveloperGuide/sqs-json-faqs.html#json-protocol-getting-started).